test/utils/asserts.hpp

```diff
@@ -8,7 +8,7 @@
 #include <gmock/gmock.h>
 #include <iosfwd>
 
-MATCHER(Traps, "")
+MATCHER(Traps, "")  // NOLINT(readability-redundant-string-init)
```

Comment:
Btw what is this error?

Reply:
It generates code like `std::string s = ""`.